full/resources/README.txt

```diff
@@ -1,48 +1,68 @@
-Overview       
-------------  
-                                                                                                              
+Overview
+------------
+
 Start using Galasa to enable deep integration testing across platforms and technologies, and run 
 repeatable, reliable, agile testing at scale across your enterprise.  
 
 This zip enables a network-free installation of Galasa, removing the requirement to connect to the 
 internet when building or running tests.
 
-This zip file contains:                                                                                                                                                                                                                        
-    - A maven directory containing dependencies that are required for building Galasa tests.
+This zip file contains:
+    - A maven directory containing dependencies that are required for running Galasa tests.
     - A javadoc directory containing javadoc API documentation for the Galasa Managers.
     - A galasactl directory containing the binaries of the Galasa CLI.
-    - An isolated.tar file - an optional Docker image that hosts the Maven directories. Use this file to host Galasa                        
-    on an internal server that can be accessed by other users.
-    - A docs.jar file that enables you to run the Galasa website locally on your machine.
-                                                                                                 
-                                                                                                                                                    
-You can find out more about Galasa on the Galasa website https://galasa.dev, or you can host the website locally by 
-running the `docs.jar` file that is contained in this zip file on your machine or on an internal server.                                                                
+    - A docs.tar file that loads a Docker image, which enables you to run the Galasa website
+      locally on your machine or on an internal server.
+    - An isolated.tar file - an optional Docker image that hosts the Maven directories.
+      Use this file to host Galasa on an internal server that can be accessed by other users.
+
+
+You can find out more about Galasa on the Galasa website https://galasa.dev, or you can host the website
+locally by following the instructions in the section below to run a web server on your machine.
+
+
+Pre-requisites
+----------------
+
+- Java 11 or above to run Galasa tests. Currently Java 21 or above is not supported.
+- Docker - If you want to load and run the Docker images provided in this zip, you will need Docker installed.
```

Comment:
Docker demon, an implementation of the docker commands (eg: From Rancher or Docker, or podman).

Reply:
Thanks, addressed in https://github.com/galasa-dev/isolated/pull/115/commits/6deb8da1580898fe2ff4735d73398c15ebe29742